modules/psap-node-feature-discovery-using-topology-updater.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_content-type: PROCEDURE
+[id="using-the-nfd-topology-updater_{context}"]
+= Using the NFD Topology Updater
+
+The Node Feature Discovery (NFD) Topology Updater is a daemon responsible for examining allocated resources on a worker node. It accounts for resources that are available to be allocated to new pod on a per-zone basis, where a zone can be a Non-Uniform Memory Access (NUMA) node. The NFD Topology Updater communicates the information to nfd-master, which creates a `NodeResourceTopology` custom resource (CR) corresponding to all the nodes in the cluster. One instance of the NFD Topology Updater runs on each node of the cluster.
```

Comment:
CRs are created only for worker nodes, should read:

 which creates a `NodeResourceTopology` custom resource (CR) corresponding to all the worker nodes in the cluster.